serializer/serializer.go

```diff
@@ -22,14 +29,20 @@ func NewSerializer(poller poller.Fetcher) *Serializer {
 
 // GetSerializedData serializes split and segment data into strings
 func (serializer *Serializer) GetSerializedData() (string, error) {
-	latestData := serializer.poller.GetSplitData()
-	if reflect.DeepEqual(latestData, poller.SplitData{}) {
-		return fmt.Sprintf(formattedLoggingScript, "{}"), nil
-	}
-
-	splitCachePreload, err := json.Marshal(latestData)
-	if err != nil {
-		return "", err
-	}
-	return fmt.Sprintf(formattedLoggingScript, string(splitCachePreload)), nil
+        latestData := serializer.poller.GetSplitData()
```

Comment:
nit: I believe the tab formatting may be off

Reply:
whoops, yep -- ty!